### PR TITLE
new data hub

### PR DESCRIPTION
**Pull request by JeremyLikness**
Initial layout for new data hub. Addresses #3833

### CODE REVIEW

File: entity-framework/dotnet-data/index.yml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            text: "Microsoft.Data.Sqlite overview"
+          - url: /sql/
+            text: "Microsoft SQL documentation"
+          - url: /azure/postgresql/flexible-server/connect-csharp
+            text: "Use C# .NET to connect and query data in Azure Database for PostgreSQL - Flexible Server"
+          - url: /azure/mysql/connect-csharp
+            text: "Use C# .NET to connect and query data in Azure Database for MySQL"
+        
+      # Card
+      - title: Document databases
```
[the remark this answers is not in the document]
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:81]: The intent is for people new to .NET/etc. They won't search for Cosmos, but rather "document database" and the one we happen to have is Cosmos.

File: entity-framework/dotnet-data/index.yml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    - title: Databases # < 60 chars (optional)
+      items:
+      # Card
+      - title: Relational databases
+        links:
+          - url: https://docs.microsoft.com/dotnet/standard/data/sqlite/
+            text: "Microsoft.Data.Sqlite overview"
+          - url: /sql/
+            text: "Microsoft SQL documentation"
+          - url: /azure/postgresql/flexible-server/connect-csharp
```
[the remark this answers is not in the document]
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:75]: that page has both Azure and on-premises docs

File: entity-framework/dotnet-data/index.yml [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+            text: "Develop an ASP.NET Core MVC web application with Azure Cosmos DB by using .NET SDK"
+          - url: /azure/cosmos-db/sql/create-notebook-visualize-data
+            text: "Create a notebook in Azure Cosmos DB to analyze and visualize the data"
+          - url: /azure/cosmos-db/mongodb/mongodb-introduction
+            text: "Azure Cosmos DB API for MongoDB"
+          - url: core/providers/cosmos/
+            text: "EF Core Azure Cosmos DB provider"
+
+      # Card
+      - title: Graph databases
```
[the remark this answers is not in the document]
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:95]: nice I'll add it

File: entity-framework/dotnet-data/index.yml [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+            text: "Overview"
+
+      # Card
+      - title: Unstructured/blob storage
+        links:
+          - url: core/querying/index.md
+            text: "Overview"
+
+      # Card
+      - title: EF Core
```
[the remark this answers is not in the document]
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:115]: great point! I'm mixed thoughts. To me discoverability is the most important and when looking at databases, seems not like a bad thing to include "this is our recommended way to connect to databases (in most scenarios).

File: entity-framework/dotnet-data/index.yml [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+            text: "Analyze it!"
+
+      # Card
+      - title: gRPC
+        links:
+          - url: core/
+            text: "Analyze it!"
+
+      # Card
+      - title: SignalR
```
[the remark this answers is not in the document]
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:194]: yes, it's one of the preferred ways in .NET to send/receive realtime/streaming data

File: entity-framework/dotnet-data/index.yml [hunk 6]
```diff
@@ -0,0 +1,216 @@
+### YamlMime:Hub
+
+title: .NET data
+summary: .NET is the open source and cross-platform for integrating data, data services and data backends with your applications.
+brand: dotnet
+
+metadata:
+  title: .NET data documentation
```
Comment (bricelam) [inline entity-framework/dotnet-data/index.yml:8]: Can we style it *.NET Data*? (everywhere) This just makes me ask, "What's data documentation?"

**Review by ajcvickers — APPROVED** [review]
This looks great! :shipit:

**Comment by JeremyLikness** [comment]
If @smitpatel is able to take a quick look, I will merge after his approval.

**Review by luisquintanilla — APPROVED** [review]
This is great @JeremyLikness. LGTM :shipit:

**Pull request merged**